src/main/java/org/bankofspring/validator/BankValidator.java

```diff
@@ -0,0 +1,30 @@
+package org.bankofspring.validator;
+
+import org.bankofspring.entities.Account;
+
+public class BankValidator {
+
+	/**
+	 * This should probably return the errors at some point.
+	 * @param fromAccount
+	 * @param toAccount
+	 * @param amount
+	 * @return
+	 */
+	public boolean validateTransfer(Account fromAccount, Account toAccount, long amount) {
+		// ensure the from account has money
+		if (fromAccount.getAccountAmount() < amount) {
+			return false;
+		}
+		
```

Comment:
validation for the toAccount not to exceed the maximum authorized amount missing?